### PR TITLE
Add mobile and phone number to register UI

### PR DESCRIPTION
**Pull request by mahdibenromdhane**

### CODE REVIEW

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]
Looks good, but I think only the Mobile phone is necessary.
You can also add a Required validator in the Mobile form control.

File: src/app/authentication/register/authentication-register.component.html [hunk 1]
```diff
@@ -42,6 +42,17 @@ <h4 class="card-title text-center">{{"authentication.register" | translate}}</h4
                       </mat-form-field>
                     </span>
                   </div>
+                  <div class="input-group-prepend">
+                    <span class="input-group-text">
+                      <mat-icon>smartphone</mat-icon>
+                      <mat-form-field class="ms-2">
+                        <input [formControl]="mobile" matInput placeholder="{{'users.mobile' | translate}}" type="text">
+                        <mat-error *ngIf="mobile.errors?.invalidPhone">
```
Comment (LucasBrazi06) [inline src/app/authentication/register/authentication-register.component.html:50]: Add generic message for required field!!!

**Review by LucasBrazi06 — CHANGES_REQUESTED** [review]

**Comment by LucasBrazi06** [comment]
@mahdibenromdhane Have you checked with Alix to implement also this feature in the mobile app?
Otherwise you will block the register in the mobile app.
Mobile App must be deployed first.

**Comment by mahdibenromdhane** [comment]
> @mahdibenromdhane Have you checked with Alix to implement also this feature in the mobile app? Otherwise you will block the register in the mobile app. Mobile App must be deployed first.

Yes ! Just checked with @AlixH, he will implement it on the mobile side.

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**